app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#F44336</color>
+    <color name="colorPrimaryDark">#D32F2F</color>
+    <color name="colorAccent">#03A9F4</color>
+    <color name="accentLight">#40C4FF</color>
+    <color name="darkFont">#10100f</color>
+    <color name="inactiveText">#9e9e9e</color>
+    <color name="done_task">#e0e0e0</color>
```

Comment:
i would rename this to the color that is is... as a rule i try and name styles and colors as close to what they are and not how I use them. because how you use them might change. so right now you're using it to represent done task but maybe in the future you use it elsewhere,.